### ISSUE TITLE
Accessibility: several color contrast issues on home page

### ISSUE THREAD

**Issue by ashleemboyer** [chakra-ui/chakra-ui-docs#1669]
### Description

Here are some color contrast issues found with both manual inspection and [axe browser extension](https://www.deque.com/axe/browser-extensions/):

- GitHub, Discord, YouTube, etc. icons in header: The color of these icons in their base state does not have sufficient contrast with the background color.
  - [WebAIM Color Contrast comparison](https://webaim.org/resources/contrastchecker/?fcolor=A0AEC0&bcolor=FFFFFF)
  - [WCAG 2.1 SC 1.4.11 Non-text Contrast (AA) details about graphical objects](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html#graphical-objects)
- "Chakra UI is a simple..." `p` text after `h1`
- "Get Started" CTA with `#ffffff` on `#319795` background
- "See showcase" CTA with `#ffffff` on `#319795` background
- "We're dedicated to..." with `#c1e0df` on `#319795` background
- "Downloads per month", "GitHub stars", etc. stats with `#ffffff` on `#319795` background
- "CHAKRA HEROES" text with `#ffffff` on `#319795` background

There are several more in the same styles and types of components, these are just a few.
